.github/workflows/release.yml

```diff
@@ -29,7 +29,6 @@ jobs:
       publish_docs: ${{ steps.check_commits.outputs.publish_docs }}
       commit_details: ${{ steps.check_commits.outputs.commit_details }}
       append_notes: ${{ steps.check_commits.outputs.append_notes }}
```

Comment:
- The PR Description feature was removed, ensure this won't affect any functionalities.